### ISSUE TITLE
Boards don't horizontally scroll back to the left

### ISSUE THREAD

**Issue by trincasidra** [mk-5/fjira#91]
**Desktop:**
 - OS: ArchLinux (6.6.7-arch1-1)
 - Terminal: kitty 0.31.0
 - Shell: bash 5.2.21
 - Architecture: x86_64
 - Fjira version: 0.10.0-1 (from [AUR](https://aur.archlinux.org/packages/fjira))

**Describe the bug**
When a board has enough columns so that they occupy more space than  the available width, the right-most visible column correctly hides midway through. However, when you move to that column, the first column disappears (ie. columns are left-aligned on the first visible column, instead of right-aligned on the last visible column).

After this, when you move back to the left-most column which is now hidden, the board won't scroll to the left. You can see which issue is highlighted since it shows on the top, and you can navigate up/down, but the column just won't show.

**To Reproduce**
1. Launch the terminal and make sure its window is narrow enough to overflow step 3 board horizontally.
2. Run fjira and open a project.
3. Open a project dashboard with multiple columns (in my case, 5).
4. Hit the right-arrow key to navigate to the right-most column.
5. Hit the left-arrow key to navigate back to the first column.

**Expected behavior**
The left-most column is visible again after moving the pointer back to it and highlighting any of its issues. Additionally, I would also expect the columns to be right-aligned, with the left-most column to just get partially hidden when moving to the right-most one if there is available space, just like how the right-most column is partially hidden when first opening the board.

**Additional notes**
Before moving:
![2023-12-18_10-36](https://github.com/mk-5/fjira/assets/15145418/d99ae5f5-1bb8-46d0-bf20-393bdd6552be)
After moving right (note that even if blurred, the selected issue is one of the first column):
![2023-12-18_10-36_1](https://github.com/mk-5/fjira/assets/15145418/af3bd557-82fa-41d0-8c70-aa61cdb48b6a)

**Comment by trincasidra** [comment]
Reviewing the releases, I noticed that my fjira version is quite outdated, and I believe the specific issue may have been fixed on the latest release thanks to [d5d3f69](https://github.com/mk-5/fjira/commit/d5d3f69c8f08808400a94b41bc58b92822528019) (fix: paginate boards).

If that is so, then my actual issue would be AUR having a very outdated version. In which case, I'll see what I can do on that end.
